e2e/rbd.go

```diff
@@ -475,7 +490,7 @@ var _ = Describe("RBD", func() {
 					framework.Failf("failed to create PVC: %v", err)
 				}
 				// validate created backend rbd images
-				validateRBDImageCount(f, 1, defaultRBDPool)
+				validateRBDImageCount(f, 11, defaultRBDPool)
```

Comment:
this is by mistake?

Reply:
i think this is only for testing, to test how the log looks like

Reply:
correct. only for testing